### ISSUE TITLE
Version 1.15.1 crashes on launch

### ISSUE THREAD

**Issue by astro** [siacs/Conversations#2200]
#### General information

* **Version:** 1.15.1, just upgraded from 1.15.0 through F-Droid
* **Device:** Moto G 2014
* **Android Version:** CyanogenMod 14.1-20161201-nightly
* **Server name:** self-hosted
* **Server software:** prosody 0.9.x
* **Installed server modules:** Stream Managment, CSI, MAM
* **Conversations source:** F-Droid

#### Steps to reproduce

1. Open conversations
2. Immediate crash

#### Expected result

Expected it not to crash.

#### Debug output

````
12-24 12:52:38.416 20509 20509 D AndroidRuntime: Shutting down VM
12-24 12:52:38.423 20509 20509 E AndroidRuntime: FATAL EXCEPTION: main
12-24 12:52:38.423 20509 20509 E AndroidRuntime: Process: eu.siacs.conversations, PID: 20509
12-24 12:52:38.423 20509 20509 E AndroidRuntime: java.lang.StringIndexOutOfBoundsException: length=0; index=-1
12-24 12:52:38.423 20509 20509 E AndroidRuntime: 	at java.lang.String.charAt(Native Method)
12-24 12:52:38.423 20509 20509 E AndroidRuntime: 	at eu.siacs.conversations.utils.UIHelper.getMessagePreview(UIHelper.java:204)
12-24 12:52:38.423 20509 20509 E AndroidRuntime: 	at eu.siacs.conversations.ui.adapter.ConversationAdapter.getView(ConversationAdapter.java:79)
12-24 12:52:38.423 20509 20509 E AndroidRuntime: 	at android.widget.AbsListView.obtainView(AbsListView.java:2370)
12-24 12:52:38.423 20509 20509 E AndroidRuntime: 	at android.widget.ListView.measureHeightOfChildren(ListView.java:1326)
12-24 12:52:38.423 20509 20509 E AndroidRuntime: 	at android.widget.ListView.onMeasure(ListView.java:1233)
12-24 12:52:38.423 20509 20509 E AndroidRuntime: 	at android.view.View.measure(View.java:19863)
12-24 12:52:38.423 20509 20509 E AndroidRuntime: 	at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:6120)
12-24 12:52:38.423 20509 20509 E AndroidRuntime: 	at android.widget.LinearLayout.measureChildBeforeLayout(LinearLayout.java:1464)
12-24 12:52:38.423 20509 20509 E AndroidRuntime: 	at android.widget.LinearLayout.measureVertical(LinearLayout.java:758)
12-24 12:52:38.423 20509 20509 E AndroidRuntime: 	at android.widget.LinearLayout.onMeasure(LinearLayout.java:640)
12-24 12:52:38.423 20509 20509 E AndroidRuntime: 	at android.view.View.measure(View.java:19863)
12-24 12:52:38.423 20509 20509 E AndroidRuntime: 	at android.support.v4.widget.SlidingPaneLayout.onMeasure(SlidingPaneLayout.java:550)
12-24 12:52:38.423 20509 20509 E AndroidRuntime: 	at android.view.View.measure(View.java:19863)
12-24 12:52:38.423 20509 20509 E AndroidRuntime: 	at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:6120)
12-24 12:52:38.423 20509 20509 E AndroidRuntime: 	at android.widget.FrameLayout.onMeasure(FrameLayout.java:185)
12-24 12:52:38.423 20509 20509 E AndroidRuntime: 	at android.view.View.measure(View.java:19863)
12-24 12:52:38.423 20509 20509 E AndroidRuntime: 	at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:6120)
12-24 12:52:38.423 20509 20509 E AndroidRuntime: 	at com.android.internal.widget.ActionBarOverlayLayout.onMeasure(ActionBarOverlayLayout.java:446)
12-24 12:52:38.423 20509 20509 E AndroidRuntime: 	at android.view.View.measure(View.java:19863)
12-24 12:52:38.423 20509 20509 E AndroidRuntime: 	at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:6120)
12-24 12:52:38.423 20509 20509 E AndroidRuntime: 	at android.widget.FrameLayout.onMeasure(FrameLayout.java:185)
12-24 12:52:38.423 20509 20509 E AndroidRuntime: 	at com.android.internal.policy.DecorView.onMeasure(DecorView.java:689)
12-24 12:52:38.423 20509 20509 E AndroidRuntime: 	at android.view.View.measure(View.java:19863)
12-24 12:52:38.423 20509 20509 E AndroidRuntime: 	at android.view.ViewRootImpl.performMeasure(ViewRootImpl.java:2270)
12-24 12:52:38.423 20509 20509 E AndroidRuntime: 	at android.view.ViewRootImpl.measureHierarchy(ViewRootImpl.java:1366)
12-24 12:52:38.423 20509 20509 E AndroidRuntime: 	at android.view.ViewRootImpl.performTraversals(ViewRootImpl.java:1619)
12-24 12:52:38.423 20509 20509 E AndroidRuntime: 	at android.view.ViewRootImpl.doTraversal(ViewRootImpl.java:1254)
12-24 12:52:38.423 20509 20509 E AndroidRuntime: 	at android.view.ViewRootImpl$TraversalRunnable.run(ViewRootImpl.java:6306)
12-24 12:52:38.423 20509 20509 E AndroidRuntime: 	at android.view.Choreographer$CallbackRecord.run(Choreographer.java:874)
12-24 12:52:38.423 20509 20509 E AndroidRuntime: 	at android.view.Choreographer.doCallbacks(Choreographer.java:686)
12-24 12:52:38.423 20509 20509 E AndroidRuntime: 	at android.view.Choreographer.doFrame(Choreographer.java:621)
12-24 12:52:38.423 20509 20509 E AndroidRuntime: 	at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:860)
12-24 12:52:38.423 20509 20509 E AndroidRuntime: 	at android.os.Handler.handleCallback(Handler.java:751)
12-24 12:52:38.423 20509 20509 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:95)
12-24 12:52:38.423 20509 20509 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:154)
12-24 12:52:38.423 20509 20509 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:6095)
12-24 12:52:38.423 20509 20509 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
12-24 12:52:38.423 20509 20509 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:886)
12-24 12:52:38.423 20509 20509 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:776)
12-24 12:52:38.427 31031  9574 W ActivityManager:   Force finishing activity eu.siacs.conversations/.ui.ConversationActivity
12-24 12:52:38.430 31031 20584 W DropBoxManagerService: Dropping: data_app_crash (2900 > 0 bytes)
````

**Comment by iNPUTmice** [comment]
This has been fixed in 1.15.2

**Comment by astro** [comment]
Nice. Now I'm just waiting for it to land in the F-Droid repo.
